CHANGELOG.md

```diff
@@ -6,6 +6,17 @@ This file follows the format suggested by [Keep a CHANGELOG](https://github.com/
 
 ## [Unreleased][Unreleased]
 
+### Added
+
+### Changed
+- [Patch] Changed icon for hamburger to solid lines.
+
+### Fixed
+- [Patch] Fix typo for "ellipsis" and "exclamation" (#6)
```

Comment:
It'd also be helpful to have the former incorrect spelling for people doing find and replace.